ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserPrompts.swift

```diff
@@ -92,13 +93,13 @@ protocol JSAlertInfo {
 
 struct MessageAlert: JSAlertInfo {
   let message: String
-  let frame: WKFrameInfo
+  let origin: URLOrigin
   let completionHandler: () -> Void
   var suppressHandler: SuppressHandler?
 
   func alertController() -> JSPromptAlertController {
     let alertController = JSPromptAlertController(
-      title: titleForJavaScriptPanelInitiatedByFrame(frame),
+      title: origin.serialized,
```

Comment:
Serialized can return nil. Can we check if: https://hackerone.com/reports/2928713 returns `nil` or `://` with this change? It shouldn't.

Reply:
Discussed in DMs, tests showed that any invalid origin wouldn't even get to this point